media-libs/opencolorio/opencolorio-2.2.1.ebuild

```diff
@@ -125,6 +126,15 @@ src_configure() {
 	cmake_src_configure
 }
 
+src_install()
+{
```

Comment:
Please don't get into habit of writing this, all ebuilds using this syntax were recently cleared to have a common style.
```suggestion
src_install() {
```

Reply:
Fixed. Thanks!